charts/ccx/scripts/ccx-yaml-gen.sh

```diff
@@ -118,554 +176,257 @@ echo "*      This configuration is specific to the CCX Dependencies system for Q
-    fi
-
-    print_colored "[optional]: Input openstack floating ip api microversion. (default FloatingIPV3) You can press \"Enter\" to skip it" "debug"
-    read floatingApiVersion
-    if [[ -z "$floatingApiVersion" ]]; then
-            echo "Using Default FloatingIPV3 version"
-            yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.floating_ip_api_version = \"FloatingIPV3\"" -i values.yaml
-    else
-            yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.floating_ip_api_version = \"$floatingApiVersion\"" -i values.yaml
-    fi
+    # Additional OpenStack configurations can be added here
+}
 
-    yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.project_id= \"$OS_PROJECT_ID\"" -i values.yaml
-
-    print_colored "REQUIRED: Input openstack floating network id. The floating_network_id refers to a floating IP pool, which is a range of public IP addresses available for assignment to virtual machines." "success"
-    read floatingNetworkId
-    yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.floating_network_id= \"$floatingNetworkId\"" -i values.yaml
-
-    print_colored "REQUIRED: Input openstack network id. The network id refers to the unique identifier assigned to a default network within the OpenStack environment" "success"
-    read networkId
-    yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.network_id= \"$networkId\"" -i values.yaml
-    
-    yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.max_jobs= 5" -i values.yaml
-    yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.retries= 5" -i values.yaml
-
-    print_colored "REQUIRED: openstack image id. we support ubuntu image 20 and 22.04 only. use openstack image list command to check id " "success"
-    read openstackImageId
-    print_colored "REQUIRED: openstack security group" "success"
-    read openstackSecurityGroup
-    openstackImageId=$openstackImageId openstackSecurityGroup=$openstackSecurityGroup yq eval '.ccx.services.deployer.config.openstack_vendors.s9s.regions.regionName |= . + { "image_id": env(openstackImageId), "secgrp_name": env(openstackSecurityGroup) }' -i values.yaml
-
-    print_colored "[optional]: Input openstack s3 endpoint for backups. You can press \"Enter\" to skip it. backup will not be enabled" "debug"
-    read openstackS3Endpoint
-    if [[ -z "$openstackS3Endpoint" ]]; then
-            echo  "Skipping backup"
-            yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.s3.nothing_here = 0" -i values.yaml
-    else
-            yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.s3.endpoint = \"$openstackS3Endpoint\"" -i values.yaml
-
-            print_colored "[Required:] Input your s3 Access Key" "success"
-            read s3AccessKey
-            yq eval ".ccx.services.deployer.config.openstack_vendors.s9s.s3.access_key = \"$s3AccessKey\"" -i values.yaml
-
```

Comment:
Hi @freddielunchbird , you deleted most of the openstack and other cloud providers,  cloud config, deployer config etc. Without the overriden yaml, it won't work. Is that intentional?

Also, These changes needs to be created on helm-ccx repo. not in this repo.